ProcSDF/GUI/Inspector.cpp

```diff
@@ -94,6 +96,24 @@ void Inspector::drawRenderingSettings()
 
 void Inspector::drawNodeGraphSettings()
 {
+	if (ImGui::Button("Add custom node"))
+	{
+		std::pair<bool, std::string> l_pickFileResult = OS::pickFile();
+		if (l_pickFileResult.first)
+		{
+			std::string l_filePath = l_pickFileResult.second;
+			CustomNode* l_customNode = new CustomNode(l_filePath);
+			if (l_customNode->isMalformed())
+			{
+				delete(l_customNode);
+			}
```

Comment:
displaying some errors would be helpful here

Reply:
we are outputting an error wherever we set the isMalformed thingie.